scripts/docker_cmake

```diff
@@ -7,6 +7,7 @@ ARGS=(
     -D BUILD_SHARED_LIBS=ON
     ### COMPILERS AND FLAGS ###
 #    -D CMAKE_CXX_FLAGS="-Wall -Wpedantic"
+    -D Trilinos_ENABLE_Fortran=ON
```

Comment:
I would move this to `docker/jenkins/build.sh`. `docker_cmake` is a good example on how to configure Trilinos with `DTK` and adding Fortran may be a problem for people using clang and not very valuable for others.